javascript/javascript2/week1/index1.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    
+    <title>Document</title>
+    <style>
+        body{
+            margin-left: 50px;
+            margin-bottom: 50px;
+
+        }
+    </style>
+</head>
+<body>
+    <div><h1>Exercise 1</h1>
+        <p>Smallest lettor in Array <br>
+           Given Array:["bil", "plante", "kaffe", "bog", "ø", "planetarium"]
+        </p>
+        <p id="smallestLetter"></p>
+
+    </div>
+    <div>
+        <h1>Exercise 2</h1>
+        <p>Danish letters count in a string<br>
+           Given String:"Blå grød med røde bær"
+        </p>
+        <p id="danishLetters"></p>
+
+    </div>
+    <div>
+        <h1>Spirit animal name generator</h1>
+        <input type="text" id="name"> <br>
+        <br><button id="getname" type="submit">Get full Name</button><br>
+        <h1 id="h1"></h1> <br>
+
+    </div>
+    <div>
+        <h1>Upgrated spirit animal name generator</h1>
+        <select name="" id="select" onchange="onChange()">
```

Comment:
Please us `addEventListener` to add the `onchange` event to the select tag. its not good practice to mix up event listeners in html and javascript.

Reply:
done